src/Raven.Client/Documents/AI/Conversation.cs

```diff
@@ -84,16 +83,22 @@ public async Task<bool> RunAsync(CancellationToken token = default)
             if (_actionRequests != null && string.IsNullOrEmpty(_userPrompt) && _actionResponses.Count == 0)
                 return false;
 
-            op = new RunConversationOperation<T>(_conversationId, _userPrompt, _actionResponses);
+            op = new RunConversationOperation<T>(_conversationId, _userPrompt, _actionResponses, _changeVector);
         }
 
         try
         {
             var r = await _aiOperations._executor.SendAsync(op, token).ConfigureAwait(false);
+            r.ChangeVector = _changeVector;
```

Comment:
Does this mean that on the second run, we'll _always_ have change vector enabled?

Reply:
Also, reading the actual code, isn't this the other way around?